### PR TITLE
Metrics upload

### PR DESCRIPTION
**Pull request by MattBuckley-TfN**
# Changes

Added script to upload the cost metrics CSVs from [OTP4GB-py](https://github.com/Transport-for-the-North/otp4gb-py) to the database.

Wait until #2 is merged before merging this into main.

# Tasks

- [ ] Have unittests been added (testing individual functions and their edge cases)
- [ ] Has documentation (including user guide) been updated
- [ ] Have new dependencies been added (or changed) in relevant `requirements.txt`
- [ ] Code linting, tests and other workflows pass

### CODE REVIEW

**Comment by MattBuckley-TfN** [comment]
This PR is awaiting confirmation of all the database changes before being merged

File: cost_upload.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+from otp4gb import routing
+from pydantic import dataclasses
+
+import scheduler
+
+##### CONSTANTS #####
+
+_NAME = pathlib.Path(__file__).stem
+LOG = logging.getLogger(_NAME)
+_CONFIG_FILE = pathlib.Path(__file__).with_suffix(".yml")
```
Comment (hulsiejames) [inline cost_upload.py:24]: Looks like this process look for a `cost_upload.yml` config file to perform this action. 

Guessing it's not included as actually contains our DB info!

Could provide an example_cost_upload.yml, but I think that could be added if we ever get interest in this or want to share. 

Can just look at the database.DatabaseConfig to know how to populate the config.

**Review by hulsiejames — DISMISSED** [review]
Looks good!

File: cost_upload.py (hunk 1 above)
Comment (MattBuckley-TfN) [reply cost_upload.py:24]: Made those changes now, please could you do a quick re-review and I'll merge everything in

**Review by hulsiejames — APPROVED** [review]
All Good!
